### PR TITLE
adding quit and ctrl-d support to influx cli

### PR DESCRIPTION
**Pull request by sebito91**
Adding in support to the influx cli for "quit" and "ctrl-d" for usability. Very open to how to handle the EOF input, seems best to leverage existing code for "exit" to ensure commands written to influx_history.

### CODE REVIEW

**Comment by otoolep** [comment]
Great stuff, badly needed. +1

@corylanou ?

File: cmd/influx/cli/cli.go [hunk 1]
```diff
@@ -180,7 +180,10 @@ func (c *CommandLine) Run() {
 			c.exit()
 		default:
 			l, e := c.Line.Prompt("> ")
-			if e != nil {
+			if e == io.EOF {
+				// Instead of die, register that someone exit the program gracefully
```
Comment (corylanou) [inline cmd/influx/cli/cli.go]: nit: `exit` -> `exited`

**Comment by corylanou** [comment]
+1

File: cmd/influx/cli/cli.go (hunk 1 above)
Comment (sebito91) [inline cmd/influx/cli/cli.go]: Just the comment, right? You OK using the ```l = "exit"``` and letting ParseCommand take it?

Comment (corylanou) [inline cmd/influx/cli/cli.go]: Yes.  The comment just didn't read right.  The code is great!

**Comment by sebito91** [comment]
Nit resolved!

**Comment by otoolep** [comment]
Thanks @sebito91

**Pull request merged**